### DISCUSSION TITLE
Conditional cell colour formatting

### DISCUSSION THREAD

**Discussion by ibsil** [andmarti1424/sc-im#824] [Q&A] [answered]
I haven't been able to find if something like this is possible. In most other spreadsheet programs, you can colour a cell conditionally based on the data in that cell. Can this be done in sc-im? For example, colouring a cell red if the number in the cell is negative, green if it's positive. I haven't had any luck finding anything in the help file that suggests this functionality but I figured I would ask anyways.

**Comment by andmarti1424** [comment, marked as answer]
Conditional formating as you would expect where you can specify ranges is not supported, I am afraid.
However, the special case of CELL_NEGATIVE is already supported on old sc and in sc-im. May be that would help you.
Take a look at CELL_NEGATIVE on doc.

**Reply by ibsil** [reply]
Hmmm, okay. The CELL_NEGATIVE case will really only be helpful in one or two situations for me. Most of the values where I would like to colour a cell is based on ranges of a result which is obviously not supported. (For example, colour the result cell blue if the numerical value falls in one range of values, yellow for another range, red for another range, white for another range, and so on). Is this a feature that may be implemented at a future date? I figured I would ask since it can come in handy with quickly identifying results.
